### DISCUSSION TITLE
After OTA no games/apps will launch

### DISCUSSION THREAD

**Discussion by degreeszero** [OnionUI/Onion#923] [Q&A]
Everything was working great then I ran the OTA update for the newest beta build and now none of the games will launch. Package manager will not launch. Retroarch does not detect and cannot load any cores. The whole system is useless.

As far as I can tell there are no emulator cores on the device, nothing installed despite having gone through the setup process. This new beta update is very broken.

I found a great solution. I reinstalled the previous beta build which works great and doesn't have any of the absurd problems the new update has.

**Comment by swologames** [comment]
I’m having the same issue after I did the OTA today on the Miyoo Mini Plus.  The about device page for mine says the firmware version is 202303262339. Does anyone have a work around or fix besides downgrading?

Thanks to the devs for great support so far! hopefully this is easy to fix.
